src/MicrosoftTeams.schema.json

```diff
@@ -157,7 +160,7 @@
         },
         "staticTabs": {
             "type": "array",
-            "description": "A set of tabs that may be 'pinned' by default, without the user adding them manually.  Static tabs declared in personal scope are always pinned to the app's personal experience.  Static tabs declared in the team scope are candidates to be pinned automatically to channels, subject to approval.",
+            "description": "A set of tabs that may be 'pinned' by default, without the user adding them manually.  Static tabs declared in personal scope are always pinned to the app's personal experience.  Static tabs declared in the team scope are candidates to be pinned automatically to channels, subject to a user's approval.",
```

Comment:
@akinolugbade @billbliss - yes, today it is Microsoft (and we'll only approve 1P apps like Assignments and Wiki etc).  In future, it might be tenant admin or even team owner/member.  After you install Evernote, we ask "do you want Evernote tab in each channel in this team?"